src/withPoints/withPoints.cpp

```
@@ -397,8 +397,51 @@ Pg_points_graph::create_new_edges() {
+                            point.vertex_id,
+                            -1,
+                            rcost_to_point};
+                        m_new_edges.push_back(rcost_edge);
+                        last_rcost = rcost_to_point;
+                    }
+                    // Update tracking variables
+                    prev_target = point.vertex_id;
+                    prev_fraction = point.fraction;
+                    agg_cost += last_cost;
+                    prev_rtarget = point.vertex_id;
+                    prev_rfraction = point.fraction;
```

Cppcheck (2.19.0)

[style] The function 'adjacent' is never used.

(unusedFunction)